client-docs/README.md

```diff
@@ -0,0 +1,363 @@
+# Mapeo Core Client API
+
+## Table of Contents
```

Comment:
would like feedback on whether this structure is hard to read through or not. taking a step back, seems okay but not great. open to ideas